src/components/Register.jsx

```diff
@@ -19,7 +20,7 @@ const Register = () => {
             <Input id="username" onChange={onRegisterChange} value={registerDetails.username} label="username"/>
             <Input id="password" onChange={onRegisterChange} value={registerDetails.password} label="password"
                    type="password"/>
-            <button>Register</button>
+            <Button name="REGISTER"/>
```

Comment:
Question, why are you using different cases for labelling your buttons? Some buttons all have upper case, some are capitalised? Why don't you follow a standard for the button names?